vortexasdk/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.32.1"
```

Comment:
The patch number should be reset to 0, no? `0.33.0`

Reply:
done